### PR TITLE
#37 Run `cargo fix --edition` & `cargo fmt` to update crate for rust 2018

### LINKED ISSUE #37: Move to 2018 edition

### PR DESCRIPTION
**Pull request by Atul9**
closes #37

### CODE REVIEW

**Comment by IsaacWoods** [comment]
Thanks!

However, I don't see changes to `Cargo.toml` - I think this should add `edition = "2018"` to the `[package]` section?

**Comment by Atul9** [comment]
@IsaacWoods Yes, we should specify the `edition = "2018"`. I have updated the PR.

File: src/madt.rs [hunk 1]
```diff
@@ -328,7 +328,7 @@ struct GicInterruptTranslationServiceEntry {
 
 pub(crate) fn parse_madt<H>(
     acpi: &mut Acpi,
-    handler: &mut H,
+    _handler: &mut H,
```
Comment (Restioson) [inline src/madt.rs]: Is there a reason for this change?

Comment (Atul9) [reply src/madt.rs]: This change is done by `cargo fix --edition` command. It may be because the field is not used.

**Comment by IsaacWoods** [comment]
Great, thanks @Atul9!

bors r+

File: src/madt.rs (hunk 1 above)
Comment (Restioson) [reply src/madt.rs]: I see

**Pull request merged**